src/components/general/MetricValue.tsx

```diff
@@ -25,18 +25,35 @@ function DashedTooltip(props: Parameters<typeof Tooltip>[0]) {
  */
 const metricValueFormatPrecision = 2
 
+interface MetricValueFormat {
+  unit: React.ReactNode
+  prefix: React.ReactNode
+  postfix: React.ReactNode
+  transform: (n: number) => number
+  formatter: (n: number) => string
+}
+
+const standardNumberFormatter = (n: number): string => `${_.round(n, metricValueFormatPrecision)}`
+const usdFormatter = (n: number): string =>
+  n.toLocaleString(undefined, { maximumFractionDigits: 2, minimumFractionDigits: 2 })
+
 /**
  * Metric Formatting Data
  */
-export const metricValueFormatData: Record<
-  string,
-  { unit: React.ReactNode; prefix: React.ReactNode; postfix: React.ReactNode; transform: (v: number) => number }
-> = {
+export const metricValueFormatData: Record<string, MetricValueFormat> = {
+  count: {
+    unit: '',
+    prefix: '',
+    postfix: '',
+    transform: identity,
+    formatter: (n) => n.toLocaleString(undefined),
```

Comment:
When running locally, ESLint is unhappy about the lack of type declarations here.

```suggestion
    formatter: (n: number): string => n.toLocaleString(undefined),
```

Reply:
interesting

Reply:
I took the liberty of committing it since I tested it locally and it's only outstanding comment. :slightly_smiling_face:

Reply:
Thanks!